dataeng/resources/warehouse-transforms.sh

```diff
@@ -24,33 +24,33 @@ then
     dbt test --models source:* --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
 fi
 
+# By default, exclude no models from testing.
+exclude_models=""
+exclude_param=""
+
+if [ "$MODEL_TAG" = 'daily' ]
+then
+    # For 'daily' models, exclude these models from being run or tested
+    exclude_models="$exclude_models finrep_map_organization_course_courserun finrep_royalty_order_dimension tag:daily-exclude"
+fi
+
+if [ "$exclude_models" != "" ]
+then
+    # If models were excluded, add the leading exclude parameter.
+    exclude_param="--exclude $exclude_models"
+fi
+
 # Compile/build all models with this tag.
-dbt run --models tag:$MODEL_TAG --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
+dbt run --models tag:$MODEL_TAG $exclude_param --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
 
 if [ "$SKIP_TESTS" != 'true' ]
 then
-    # By default, exclude no models from testing.
-    exclude_models=""
-    exclude_param=""
-
     if [ "$TEST_SOURCES_FIRST" = 'true' ]
     then
         # Exclude the sources from testing, since they were already tested pre-'dbt run'.
         exclude_models="$exclude_models source:*"
     fi
 
-    if [ "$MODEL_TAG" = 'daily' ]
-    then
-        # For 'daily' models, exclude these two models from finrep that inexplicably are tested when testing those models.
-        exclude_models="$exclude_models finrep_map_organization_course_courserun finrep_royalty_order_dimension tag:daily-exclude"
-    fi
-
-    if [ "$exclude_models" != "" ]
-    then
-        # If models were excluded, add the leading exclude parameter.
-        exclude_param="--exclude $exclude_models"
-    fi
```

Comment:
I think this needs to be restored, so that the TEST_SOURCES_FIRST exclusion above still gets into the exclude_param as well as the exclude_models.   It could of course be restored in the same if statement (i.e. on line 52).    But the code below only uses exclude_param.

Reply:
I think this is fixed now.